tools/ui-components/src/alert/alert.tsx

```diff
@@ -27,7 +27,7 @@ export const Alert = ({
   const variantClass = variantClasses[variant];
 
   const classes = [
-    'p-4 mb-6 border border-solid border-1 break-words',
+    'px-4 py-1 mb-6 border border-solid border-1 break-words',
```

Comment:
Although flash uses the alert component, they have different styles and different paddings.
If the override is removed, the alerts that have elements with margin button will have an uneven spacing.

Here are a few avenues we could explore:
1. keep the re-write for this particular case
2. create two different components
3. pass prop that adjust the padding
4. combine flash and alert into one component

Going with number 1 for now seems to be the most straight forward option.
Curious to know what you think.